draft-birkholz-scitt-receipts.md

```diff
@@ -128,17 +154,24 @@ The following parameters MUST be included in the protected header of the counter
 
 # CCF 2 Tree Algorithm
 
-The CCF 2 tree algorithm corresponds to the algorithm implemented in the CCF version 2 framework.
+The CCF 2 tree algorithm documents the algorithm based on a binary Merkle tree over the sequence of all ledger entries that is implemented in the CCF version 2 framework.
+
+> Add pointer to CCF v2? 
 
 ## Additional Parameters        {#parameters}
 
-The following additional service parameters are defined:
+The algorithm requires that the TS define 
+additional parameters:
 
-Hash Algorithm: The hash algorithm used for the Merkle Tree (see {{hash-alg-registry}}).
+- Hash Algorithm: The hash algorithm used in its Merkle Tree (see {{hash-alg-registry}}).
 
-Signature Algorithm: The signature algorithm used (see {{sig-alg-registry}}).
+- Signature Algorithm: The signature algorithm used (see {{sig-alg-registry}}).
 
-Service Certificate: The X.509 root certificate (self-signed) used as trust anchor to verify signatures generated by the transparency service.
+- Service Certificate: The X.509 certificate used as trust anchor to verify signatures generated by the transparency service.
+
+The Service Certificate MUST be consistent with the Service identifier and Signature Algorithm.  
+
+> No need to require that it be self-signed, right? 
```

Comment:
We do require it, discussed with @ad-l recently. Not making it the root would complicate things too much. Also, the cert is not changing over the lifetime, which is typially not the case for intermediate certs.

Reply:
Why not take a TS certificate chain then, with MAY be self signed?